modules/collectd/templates/etc/collectd/conf.d/postgresql.conf.erb

```diff
@@ -37,6 +37,16 @@ LoadPlugin postgresql
     </Result>
   </Query>
 
+  <Query publishing_api_content_item_count>
+    Statement "SELECT count(*) AS count \
+               FROM publishing_api_production.content_items;"
```

Comment:
Does this work? The connection will be to the `postgres` database. I don't think you can select across databases like this in postgres?
```
postgres=# SELECT count(*) AS count FROM publishing_api_production.content_items;
ERROR:  relation "publishing_api_production.content_items" does not exist
```